src/tests/ftest/util/ior_test_base.py

```diff
@@ -102,13 +102,15 @@ def get_job_manager_command(self):
         mpirun_path = os.path.join(mpio_util.mpichinstall, "bin")
         return Mpirun(self.ior_cmd, mpirun_path)
 
-    def run_ior(self, manager, processes):
+    def run_ior(self, manager, processes=None):
         """Run the IOR command.
 
         Args:
             manager (str): mpi job manager command
             processes (int): number of host processes
         """
+        if self.processes and processes is None:
+            processes = self.processes
```

Comment:
Why wouldn't we just pass in self.processes into this method?

Reply:
We can't do that as nvme_io.py is using different set of processes for different test case. It will break that test.